libraries/rush-lib/src/logic/base/BaseInstallManagerTypes.ts

```diff
@@ -82,10 +82,28 @@ export interface IInstallManagerOptions {
   maxInstallAttempts: number;
 
   /**
-   * Filters to be passed to PNPM during installation, if applicable.
-   * These restrict the scope of a workspace installation.
+   * If `filteredProjects` is `undefined`, then Rush will perform a regular `pnpm install`.
+   *
+   * If `filteredProjects` is specified, then Rush will perform a "filtered install" by invoking PNPM like this,
+   * a special mode which installs or updates only a subset of the lockfile:
+   *
+   * `pnpm install --filter project1 --filter project5 --filter project6`
+   *
+   * If `filteredProjects` is specified and the set is empty, then the install manager will skip installation entirely.
+   *
+   * @remarks
+   * If `filteredProjects` specifies all projects in the workspace or subspace, filtering will still occur;
+   * it is the caller's responsibility to optimize that.
```

Comment:
@iclanton Why do you perform the `filteredProjects?.size === selectedSubspace.getProjects().length ? undefined : filteredProjects;` optimization in the caller instead of in `WorkspaceInstallManager.pushConfigurationArgs()`?

Reply:
Could easily do it there.